examples/demo-app/src/analytics.js

```diff
@@ -0,0 +1,53 @@
+import { ActionTypes } from "kepler.gl/actions";
```

Comment:
Please confit your editor settings to get rid of extra space in the bracket

Reply:
Moving the .prettierrc to the root directory fixes this - if it's okay to do that